Leaflet.Graticule.js

```diff
@@ -192,7 +200,17 @@ L.LatLngGraticule = L.Layer.extend({
             return this.options.latFormatTickLabel(lat);
         }
 
+
```

Comment:
Also! Please open this on another branch name, since we'll want to submit it as a PR to the parent https://github.com/Leaflet/Leaflet.Graticule once it's ready! `preserve-decimal-precision` is a good branch name?